webpack/webpack.common.js

```diff
@@ -261,31 +261,39 @@ module.exports = () => {
 
         resolve({
             entry: {
-                ...archesCoreEntryPointConfiguration,
-                ...archesApplicationsEntrypointConfiguration,
-                ...projectEntryPointConfiguration,
+                ...entryPoints,
                 ...CSSFilepathLookup,
             },
             output: {
-                assetModuleFilename: 'img/[name].[contenthash][ext]',
-                filename: '[name].[contenthash].js',
+                assetModuleFilename: '[name].[contenthash][ext]',
+                chunkFilename: 'chunks/[name].[contenthash].js',
                 clean: true,
+                filename: '[name].[contenthash].js',
                 path: Path.resolve(__dirname, APP_ROOT, 'media', 'build'),
                 publicPath: STATIC_URL,
             },
             optimization: {
+                moduleIds: 'deterministic',
```

Comment:
further chunk optimization to improve caching ( and therefore rebuild times -- especially prudent for the webpack development server )